### ISSUE TITLE
Could not find a file named "pubspec.yaml"

### ISSUE THREAD

**Issue by sketchbuch** [myConsciousness/freezer#17]
When I run 
`dart run freezer:main`

I get an error:
`Could not find a file named "pubspec.yaml" in "/home/sketchbuch/.pub-cache/hosted/pub.dartlang.org/_fe_analyzer_shared-46.0.0".`

I'm guessing I have done something wrong as I'm quite new to flutter/dart, but do you know what the cause of the error could be?

**Comment by myConsciousness** [comment]
Hi @sketchbuch , thanks for trying this library! :)

Hm, I have looked at the error message, and at first glance it appears that there is something in the cache of your local environment that is causing the problem. At least, I installed and tried this library and did not reproduce this error.

Did you already try `flutter clean` or delete `pubspec.lock` and exexute `flutter pub get` again? I'm still searching why it happens but there are some cases:

https://github.com/dart-lang/sdk/issues/49167
